.pylintrc

```diff
@@ -91,7 +91,6 @@ enable=import-error,
 # 2. See if there's spurious ones; if so disable.
 # 3. Record above.
 # 4. Remove from this list.
-       # deprecated-method,
```

Comment:
Why'd you delete it without enabling it? Local testing suggests it's not too noisy so should be useful.